Expense Tracker/script.py

```diff
@@ -0,0 +1,97 @@
+from tkinter import ttk
+from tkinter import messagebox
+from tkinter import *
+from tkinter.ttk import Notebook
+from tkcalendar import DateEntry
+import sqlite3
+
+
+def Addexpense():
+    x = Edate.get()
+    y = Title.get()
+    z = Eexpense.get()
+    data = [x,y,z]
+    TVExpense.insert('','end',values=data)
+    with db:
+        c = db.cursor()
+        c.execute("INSERT INTO expense(Dates, Items, Expense) VALUES(?,?,?)",(x,y,z))
```

Comment:
handle the case when there is no expense table.

Reply:
If there is no table then it creates automatically @kaustubhgupta